app/javascript/components/textual_summary/table_list_view.jsx

```diff
@@ -28,21 +28,39 @@ const renderRow = (row, i, colOrder, rowLabel, onClick) => (
 export default function TableListView(props) {
   const { headers, values, title } = props;
 
-  /** Function to generate rows for structured list. */
-  const miqListRows = (list) => {
-    const headerKeys = props.headers.map((item) => (item.key));
-    return list.map((item) => (headerKeys.map((header) => item[header])));
+  /** Function to generate heders/rows for the default structured list. */
+  const miqListDefaultTable = () => {
+    const data = [];
+    values.map((item) => data.push({ ...item, label: item.name }));
```

Comment:
`const data = values.map((item) => ({ ...item, label: item.name }));`

Reply:
@DavidResende0 see above... anything else needed for this PR?

Reply:
`data` is getting the values pushed to it within the map function so it's correct as is. I discussed with @jeffibm separately and the only thing left is to squash the commits.

Reply:
Sounds good, let us know when it's ready to go!  🚀